### ISSUE TITLE
hyper-parameter tuning, how to reproduce the results on the informer paper?

### ISSUE THREAD

**Issue by BumjunJung9287** [zhouhaoyi/Informer2020#341]
Hi I am Bumjun Jung, a master’s student of Harada laboratory in the University of Tokyo.

Currently I am working on developing a memory-efficient self-attention mechanism on Transformer architecture.

I am doing an experiment on the ECL, WTH, ETT-small datasets that you have provided in github. 
However, the experiment with your codes did not match the Test MSE loss reported in your paper.. 

Could you let me know how I should fix the hyper-parameter on your codes to match the test MSE loss reported in your Informer paper?

Also, I think the MSE loss in the paper used ETT-large or ETT-full datasets, is it correct?
If so, is it possible for me to access ETT-large or ETT-full datasets so that I can do the comparison with your methods?

Thank you for reading such long questions.
I would very appreciate it if you answer the questions.

**Comment by zhouhaoyi** [comment]
Please refer to the scripts folder, which contains the hyperparameters. We have also included a section, named Reproducibility.

We are only with the ETT-small, the other two datasets have not been released.

**Comment by BumjunJung9287** [comment]
I see, thank you for the kind answer.
I will follow the Reproducibility section.

About the ETT-small data, is it correct that ETT-small was the data that is used in Informer paper? or are the other two datasets used?

**Comment by BumjunJung9287** [comment]
Thank you so much!
